e2e/testdata/fn-eval/fn-source/exclude-meta-resources-by-default/.expected/exec.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+
+kpt fn source \
+| kpt fn eval - --image gcr.io/kpt-fn/set-namespace:v0.1.3 -- namespace=staging \
+| kpt fn sink .
```

Comment:
Mentioned in another PR, but we need to test both:

a) kpt fn source HERE | ... | kpt fn sink HERE
b) kpt fn source HERE | ... | kpt fn sink THERE

Currently, they content of HERE and THERE would be different.

Reply:
out-of-place sink is tested in `e2e/testdata/fn-eval/out-of-place`. Do you mean we need to test both in-place and out-of-place for every scenario?

Reply:
There are 4 test cases to be tested here:
case `a` (`in-place`):

- default where `--include-meta-resources` is `false`
- `--include-meta-resources` is `true`

case `b` (`out-of-place`):
- default where `--include-meta-resources` is `false`. (here `out-of-place-dir` must not contain meta resources
- `--include-meta-resources` is `true` (here `out-of-place-dir` must contain meta resources

Are all the four covered ?

Reply:
@droot That means we need to test in-place and out-of-place for *include meta resources* feature. My question is if we need to test all these scenarios, do we need to test in-place and out-of-place for all other cases as well? Like cases about function config and function chain etc.

Reply:
Anyway, I added more test cases for including meta resources